python/pyspark/context.py

```diff
@@ -173,6 +173,7 @@ def _do_init(self, master, appName, sparkHome, pyFiles, environment, batchSize,
             self._jvm.PythonAccumulatorParam(host, port))
 
         self.pythonExec = os.environ.get("PYSPARK_PYTHON", 'python')
+        self.pythonVer = sys.version_info.major * 10 + sys.version_info.minor
```

Comment:
Probably not a huge deal, but it might be better to pass major and minor as separate parameters. You know, in case python 2.10 ever comes out. :-)

Or, at least, use a larger multiplier.